src/libslic3r/GCode.cpp

```diff
@@ -3715,8 +3715,12 @@ std::string GCode::_extrude(const ExtrusionPath &path, std::string description,
 #endif
         } else if (m_config.get_abs_value("bridge_acceleration") > 0 && is_bridge(path.role())) {
             acceleration = m_config.get_abs_value("bridge_acceleration");
+        } else if (m_config.sparse_infill_acceleration.value > 0 && (path.role() == erInternalInfill)) {
+            acceleration = m_config.sparse_infill_acceleration.value;
```

Comment:
We need to use `m_config.get_abs_value("sparse_infill_acceleration")` to get correct value